simde/wasm/simd128.h

```diff
@@ -3115,20 +3127,10 @@ simde_wasm_f32x4_abs (simde_v128_t a) {
       r_.neon_f32 = vabsq_f32(a_.neon_f32);
     #elif defined(SIMDE_POWER_ALTIVEC_P6_NATIVE)
       r_.altivec_f32 = vec_abs(a_.altivec_f32);
-    #elif defined(SIMDE_VECTOR_SUBSCRIPT_SCALAR)
-      int32_t SIMDE_VECTOR(16) m = HEDLEY_REINTERPRET_CAST(__typeof__(m), a_.f32 < SIMDE_FLOAT32_C(0.0));
```

Comment:
Can this be restored, using `simde_math_signbit`?

Reply:
I'm afraid I couldn't think of an obvious way -- signbit takes a scalar so it can't just be substituted in place of the `a_.f32 < 0.0` test. We could run signbit on each lane individually, but at that point I don't think it's going to be better than the (hopefully vectorized) for loop below it. :-/ SIMDe does have SSE2/Neon/Altivec versions of this one at least.